types/test.d.ts

```diff
@@ -118,6 +118,30 @@ export interface Project<TestArgs = {}, WorkerArgs = {}> extends ProjectBase {
 
 export type FullProject<TestArgs = {}, WorkerArgs = {}> = Required<Project<TestArgs, WorkerArgs>>;
 
+export type WebServerConfig = {
+  /**
+   * Shell command to start the webserver. For example `npm run start`.
+   */
+  command: string,
+  /**
+   * The port that your server is expected to appear on. If not specified, it does get automatically collected via the 
+   * command output when a localhost URL gets printed.
+   */
+  port?: number,
+  /**
+   * Environment variables which get set during the execution. Default it will use the environment variables from the current process.
+   */
+  env?: Record<string, string>
```

Comment:
- Missing comma
- I think env also takes booleans and numbers? Check Node's types

Reply:
They are using `NodeJS.ProcessEnv` which is `[key: string]: string | undefined` so I think we are good with it, `undefined` values are ignored referring to their docs.